lib/web_ui/dev/driver_manager.dart

```diff
@@ -20,7 +20,34 @@ import 'utils.dart';
 ///
 /// This manager can be used for both macOS and Linux.
 class ChromeDriverManager extends DriverManager {
-  ChromeDriverManager(String browser) : super(browser);
+
+  /// Directory which contains the preferred version of Chrome.
+  ///
+  /// On LUCI we are using the CIPD packages to control Chrome binaries we use.
+  /// There will be multiple CIPD packages loaded at the same time. Yaml file
+  /// `driver_version.yaml` contains the version number we want to use.
+  ///
+  /// Local integration tests are still using the system Chrome. This should be
+  /// fixed in issue: https://github.com/flutter/flutter/issues/53179.
```

Comment:
This should be a TODO rather than a doc comment

Reply:
done.